placeholder.json

```diff
@@ -278,6 +278,13 @@
     {
       "Key": "tooltip copied text",
       "Text": "Copied!"
+    },    {
```

Comment:
not sure if this is needed for Mack

Reply:
since the fields are coming I guess we can keep them for future implementation